### DISCUSSION TITLE
[Feature] Encryption

### DISCUSSION THREAD

**Discussion by cinderblockgames** [immich-app/immich#2830] [Feature Request]
### The feature

If I let others onto my instance, it would be nice to be able to assure them that I can't access their photos.

Obviously, this can wait until the app is a bit more stable, since having them plaintext is good for recovery if things change too quickly. 

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #450
